xbmc/filesystem/DirectoryCache.h

```diff
@@ -55,10 +58,9 @@ namespace XFILE
     void ClearCache(std::set<std::string>& dirs);
     void CheckIfFull();
 
-    std::map<std::string, CDir*> m_cache;
-    typedef std::map<std::string, CDir*>::iterator iCache;
-    typedef std::map<std::string, CDir*>::const_iterator ciCache;
-    void Delete(iCache i);
+    std::map<std::string, CDir> m_cache;
+    typedef std::map<std::string, CDir>::iterator iCache;
+    typedef std::map<std::string, CDir>::const_iterator ciCache;
```

Comment:
While you're at it, can you change `typedef` to `using`?

Reply:
Sure! I also applied the formatting changes suggest by Jenkins.

Reply:
i thought about removing the typedefs and replacing `iCache/ciCache` with `auto/const auto`.
but it‘s up to you :)